src/main/scala/io/iohk/ethereum/consensus/ethash/difficulty/EthashDifficultyCalculator.scala

```diff
@@ -5,15 +5,21 @@ import io.iohk.ethereum.domain.BlockHeader
 import io.iohk.ethereum.utils.BlockchainConfig
 
 class EthashDifficultyCalculator(blockchainConfig: BlockchainConfig) extends DifficultyCalculator {
-  import blockchainConfig.{difficultyBombContinueBlockNumber, difficultyBombPauseBlockNumber, homesteadBlockNumber, difficultyBombRemovalBlockNumber}
+  import blockchainConfig.{
+    difficultyBombContinueBlockNumber,
+    difficultyBombPauseBlockNumber,
+    homesteadBlockNumber,
+    difficultyBombRemovalBlockNumber,
+    byzantiumBlockNumber
+  }
 
   val DifficultyBoundDivision: Int = 2048
   val FrontierTimestampDiffLimit: Int = -99
   val ExpDifficultyPeriod: Int = 100000
   val MinimumDifficulty: BigInt = 131072
+  val RelaxDifficulty: BigInt = BigInt(10).pow(6) * 3
```

Comment:
I think it would be better to write it as number, otherwise scalastyle can complain on magic numbers ( here it does not, but in `BlockRewardCalculator` it is)

Reply:
i've used it to not make a mistake with zeros but I'll do as you proposed